src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -300,7 +331,7 @@ class ColorPickerDialog extends PureComponent {
       <Dialog
         visible={visible} //TODO: pass all Dialog props instead
         width="100%"
-        height={null}
+        height={undefined}
         bottom
```

Comment:
I remember passing `null` to Dialog's height has different outcome than `undefined`. Why did you change it?

Reply:
It's actually meaningless now and I'll remove the prop entirely. If I remember correctly, the point of passing null was to get the dialog to take the content height. Now we can just not pass height. The only way to change the Dialog's height is from 'dialogProps', which should be deprecated.

Reply:
Ok great.